examples/add_column.md

```diff
@@ -0,0 +1,92 @@
+# df.add_column()
```

Comment:
@szuckerman I'm super happy that you've added examples! 

Can I ask, how open are you to formatting this not as a Markdown file, but instead as a ReStructured Text file under `docs/`? The RST files are built as part of the hosted documentation on ReadTheDocs, and it'll get more visibility. 

That said, if it's too much of a burden, I'm happy to delegate or handle the work for a future PR, and accept this one in the interim.

Reply:
Sure; I'm fine with the RST formatting. I'll update accordingly. 

Are you ok with the structure of the examples? I'm thinking I should update to adding the parameter values in the header similar to the Pandas documentation (instead of having empty parentheses).